app/src/main/java/org/schulcloud/mobile/data/local/PreferencesHelper.java

```diff
@@ -70,6 +70,15 @@ public String saveCurrentUsername(String username) {
         return null;
     }
 
+    public String saveCurrentSchoolId(String schoolId) {
+        SharedPreferences.Editor editor = mPref.edit();
+        editor.putString("currentSchool", schoolId);
+
+        if (editor.commit())
+            return schoolId;
+        return null;
```

Comment:
This is my first bigger contribution to the project, so I adapted to the existing structure. The error handling is done like that everywhere else in the file:
https://github.com/schul-cloud/schulcloud-mobile-android/blob/01178ff91828e9b5d76f20ecb4b1cc6a0e79f605/app/src/main/java/org/schulcloud/mobile/data/local/PreferencesHelper.java#L37-L41
https://github.com/schul-cloud/schulcloud-mobile-android/blob/01178ff91828e9b5d76f20ecb4b1cc6a0e79f605/app/src/main/java/org/schulcloud/mobile/data/local/PreferencesHelper.java#L54-L58
https://github.com/schul-cloud/schulcloud-mobile-android/blob/01178ff91828e9b5d76f20ecb4b1cc6a0e79f605/app/src/main/java/org/schulcloud/mobile/data/local/PreferencesHelper.java#L66-L70
https://github.com/schul-cloud/schulcloud-mobile-android/blob/01178ff91828e9b5d76f20ecb4b1cc6a0e79f605/app/src/main/java/org/schulcloud/mobile/data/local/PreferencesHelper.java#L86-L90
https://github.com/schul-cloud/schulcloud-mobile-android/blob/01178ff91828e9b5d76f20ecb4b1cc6a0e79f605/app/src/main/java/org/schulcloud/mobile/data/local/PreferencesHelper.java#L97-L101
https://github.com/schul-cloud/schulcloud-mobile-android/blob/01178ff91828e9b5d76f20ecb4b1cc6a0e79f605/app/src/main/java/org/schulcloud/mobile/data/local/PreferencesHelper.java#L108-L112
https://github.com/schul-cloud/schulcloud-mobile-android/blob/01178ff91828e9b5d76f20ecb4b1cc6a0e79f605/app/src/main/java/org/schulcloud/mobile/data/local/PreferencesHelper.java#L119-L123